north/src/manifest.rs

```diff
@@ -160,15 +218,22 @@ pub enum MountFlag {
 }
 
 #[derive(Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct DirMount {
+    pub host: Option<std::path::PathBuf>,
+    pub flags: Option<HashSet<MountFlag>>,
+}
+
+#[derive(Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum MountSource {
+    Resource(Resource),
```

Comment:
Think there's a third type missing: `Persist`. The container specific (optionally rw) dir we create for the app.